<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AJ Bank | Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/Login.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginB">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>627</y>
      <width>332</width>
      <height>75</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Confirm.png);
background-position:center;border-radius:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginUsernameT">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginPasswordT">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/White.png);font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>93</y>
      <width>150</width>
      <height>581</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Back.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SuggestButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>0</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Suggest.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMessage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>841</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
